src/util.rs

```diff
@@ -22,9 +22,19 @@ pub fn path_to_bytes<P: AsRef<Path>>(path: &P) -> Option<&[u8]> {
 }
 
 pub fn get_db() -> Result<DB> {
-    let mut path = config::zo_data()?;
-    path.push("db.zo");
-    DB::open(path)
+    let data_dir = config::zo_data_dir()?;
+    let db_file = data_dir.join("db.zo");
```

Comment:
Avoidable `join` - try replacing it with a `push`.

Reply:
Forgot to go through the other part and replace these joins, whoops.